libs/blocks/modal/modal.js

```diff
@@ -175,6 +176,7 @@ export async function getModal(details, custom) {
       .forEach((element) => element.setAttribute('aria-disabled', 'true'));
   }
   if (dialog.classList.contains('commerce-frame')) {
+    document.body.classList.add('modal-open');
```

Comment:
If this is commerce-specific behaviour, shouldn't the class naming reflect that in some way? The generic _modal-open_ naming would lead me to believe that this class is there whenever any modal is displayed.

Reply:
update the css class name